tests/unit/components/card/ComponentDefinitionCard.spec.js

```diff
@@ -135,13 +139,25 @@ describe('Test component: ComponentDefinitionCard', () => {
   });
 
   describe('Test function: onClickItem', () => {
-    it('should call addNewComponent when isTemplate is false', () => {
-      const addNewComponent = jest.fn();
-      PluginManager.addNewComponent.mockImplementation(addNewComponent);
+    it('should call addNewComponent with default path when isTemplate is false', async () => {
```

Comment:
nothing in the test specifies that `isTemplate is false`.
=> rename or specify

Reply:
done